### PR TITLE
channels list, add channel 

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

https://app.clickup.com/t/6a5vb0
https://app.clickup.com/t/6a5vaz

PR intended to be tested with API branch:  https://github.com/mirumee/saleor/pull/5789

### Screenshots
Screenshot 2020-07-07 at 13 36 44
Screenshot 2020-07-07 at 13 37 38
Screenshot 2020-07-07 at 13 47 40

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [CU-6a5vb0 User needs to be able to access channels section of Configuration](https://app.clickup.com/t/6a5vb0)

**Comment by patrys** [comment]
Task linked: [CU-6a5vaz User needs to be able to create additional channel](https://app.clickup.com/t/6a5vaz)

File: src/channels/mutations.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import makeMutation from "@saleor/hooks/makeMutation";
+import gql from "graphql-tag";
+
+import { ChannelCreate, ChannelCreateVariables } from "./types/ChannelCreate";
+
+export const channelErrorFragment = gql`
```
Comment (orzechdev) [inline src/channels/mutations.ts:6]: Let's move fragments to `src/fragments/channels.ts` to be consistent with the latest changes made in https://github.com/mirumee/saleor-dashboard/pull/592

**Comment by krzysztofwolski** [comment]
Design: 
image

Storybook:
image


Without search/filters, there is no use for tabs

**Pull request merged**